products/checkbook/build_scripts/build.py

```diff
@@ -141,8 +140,14 @@ def _assign_checkbook_category(df: pd.DataFrame, sql_dir=SQL_QUERY_DIR) -> pd.Da
     df["bc_category"] = None
     df["cp_category"] = None
 
-    df.to_sql("capital_projects", ENGINE, if_exists="replace", index=False)
-    with ENGINE.connect() as conn:
+    df.to_sql(
+        "capital_projects",
```

Comment:
Seems like this might have broken something? Error now that "capital_projects" table doesn't exist in